### PR TITLE
got the first line

### PR DESCRIPTION
**Pull request by minaa12**

### CODE REVIEW

**Review by gigamonkey — APPROVED** [review]
Okay. Go ahead and merge this but don't delete the branch. Then you can make a new PR with your next set of changes and I'll be able to review just the new code.

File: projects/tic-tac-toe/public/script.js [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+drawText('o', 615, 300, 'red', 200)
+
+drawText('x', 615, 140, 'black', 200)
+drawText('o', 615, 140, 'red', 200)
+
+drawText('x', 615, 470, 'black', 200)
+drawText('o', 615, 470, 'red', 200)
+
+drawText('x', 615, 470, 'black', 200)
+drawText('o', 615, 470, 'red', 200)
```
Comment (gigamonkey) [inline projects/tic-tac-toe/public/script.js:20]: This is fine for seeing how things work but you should also start thinking about how to position the marks in response to user clicks. See the DOM Events slides for some examples. In particular this slide is close to what you need:

https://intro.gigamonkeys.com/slides/dom-events/#/5/0/3

**Review by gigamonkey — APPROVED** [review]
You should probably start making a list of the different things you need to get done by Friday. You have some of the basics here but you also have quite a few parts yet that you need to work out.

File: projects/tic-tac-toe/public/script.js (hunk 1 above)
Comment (minaa12) [reply projects/tic-tac-toe/public/script.js:20]: i will take a look

**Pull request merged**